alma/alma-woocommerce-gateway
# feat: change client fees wording in BO

## Reason for change

Wording change due to regulations

[Linear task](https://linear.app/almapay/issue/MPP-979/clarify-usury-rate-impact-on-customer-fees-displayed)

### How to test

Go to the BO of the plugin alma and check the wording in en and fr

![Capture d’écran du 2023-11-15 12-59-21](https://github.com/alma/alma-woocommerce-gateway/assets/110386752/5a86852f-5bc4-41e4-868c-9a7f372a3761)
![Capture d’écran du 2023-11-15 12-58-30](https://github.com/alma/alma-woocommerce-gateway/assets/110386752/e2ce24e7-9802-42a7-a13e-e17fee539c23)
